### ISSUE TITLE
Review if we can use `fsspec` for data loading

### ISSUE THREAD

**Issue by roll** [frictionlessdata/frictionless-py#1462]
# Overview

https://pypi.org/project/fsspec/

**Comment by davidgasquez** [comment]
Heya @roll! I'm curious if you can expand on this one. Are you thinking on using `fsspec` to replace the `scheme` parts of the codebase? 

I've been thinking on the best way to read/write data on a non existing scheme and thinking about this these days.

**Comment by roll** [comment]
@davidgasquez 
Yes, we historically used these loaders approach created for `tabulator`, but I'm curious if `fsspec` can handle it (I haven't dove into it yet unfortunately)

**Comment by davidgasquez** [comment]
I think it should! 

A potentiall intermediary approach might be to add a `fsspec` scheme via plugins. Does this makes sense?

**Comment by roll** [comment]
TBH, currently, I'm not sure as it seems `fsspec` might just a perfect replacement for the whole Loaders part making the parsers and the framework way simpler.

We're going to work soon on the v6 - https://github.com/orgs/frictionlessdata/projects/16/views/11 - which will include optimizations for the whole reading flow. Might be a good chance for the `fsspec` migration

**Comment by martindurant** [comment]
Happy to help!
